action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
   cosign-release:
     description: 'cosign release version to be installed'
     required: false
-    default: 'v1.13.1'
+    default: 'v2.0.0-rc.0'
```

Comment:
does it make sense to default to a RC build?

not sure if users will expect the default to be a release candidate. i think it we should support letting users specify RC builds if they choose do so, but not default them into it.
and similarly for the bootstrap version, we should choose a non-RC version for it imho.

Reply:
I just wanted to change the code to point to latest available version.

Reply:
I'll revert it, but I feel once we get our v2 stable version ready, we should change it.

Reply:
agree, once v2 stable version, we should change it to that.

thanks